vsc_pdf_template_transformer/lib/models/tpl_pdf_page_format.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/pdf.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+
+part 'tpl_pdf_page_format.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplPdfPageFormat {
+  TplPdfPageFormat(
+      {this.width = 0,
```

Comment:
Don't initialize values in the constructor, and hence don't make the fields `final`. If you initialize them, they will become the defaults if not specified in the JSON.